plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InviteCommand.java

```diff
@@ -42,14 +53,52 @@ public void execute(@CanManage User deputy, Guild guild, String[] args) {
         when(args.length < 1, this.messages.generalNoNickGiven);
         when(guild.getMembers().size() >= this.config.maxMembersInGuild, formatter.format(this.messages.inviteAmount));
 
-        User invitedUser = UserValidation.requireUserByName(args[0]);
+        if (args[0].equals(this.config.inviteCommandAllArgument)) {
+
+            Double range;
+
+            try {
+                range = Math.pow(Double.parseDouble(args[1]), 2);
+            }
+            catch (Exception exception) {
+                range = this.config.inviteCommandAllMaxRange;
+            }
+
+            Double finalRange = range;
```

Comment:
```suggestion
            double range = Option.attempt(NumberFormatException.class, () -> Math.pow(Double.parseDouble(args[1]), 2))
                    .orElseGet(this.config.inviteCommandAllMaxRange);
```

Reply:
https://github.com/FunnyGuilds/FunnyGuilds/pull/2024#discussion_r906854254
`orThrow(new ValidationException())` wysyła wiadomość do gracza

Reply:
problem jest z tym, że to może jeszcze wywalić OutOfBoundException

Reply:
że w `args[1]`? jak tak to sprawdź długość array

Reply:
No albo zrobimy Exception.class i problem z głowy

Reply:
proszę tak nie robić

Reply:
I tak jak pisałem wiadomość typu: Zapraszam graczy w pobliżu xxx bloków

Reply:
> proszę tak nie robić 

No to jak? Default jest nawet jeżeli argumentu nie ma (jeżeli jest /zapros * to ma być default ale jak jest zapros * 100 to ma być 100 ale jak jest /zapros * aaaa to ma być default)

Reply:
> prawdź długość array

Reply:
No sprawdzę i powiedzmy, że będzie 1. Co wtedy? Range dalej ma być wartością z configu, więc nie rozumiem jak chciałbyś to zrobić

Reply:
> więc nie rozumiem jak chciałbyś to zrobić
nic nie mówiłem o tym jak masz rozplanować logikę tego jak ktoś wyśle za mało argumentów, napisałem tyle żebyś nie handlował całego Exception dla tablicy tylko sprawdził długość argumentu. `if (args.lenght <= 1) {`